ddl/ddl_api.go

```diff
@@ -868,6 +868,7 @@ func checkColumnValueConstraint(col *table.Column) error {
 
 func checkDuplicateColumn(cols []interface{}) error {
 	colNames := set.StringSet{}
+	colNames.Insert("_tidb_rowid")
```

Comment:
Mysql does not have this behavior. Please add a comment about why this code was added. 
How about the operation of `alter table ... add column _tidb_rowid`?

Reply:
@zimulala Thanks for pointing out the problem. I will fix it soon.